# This expresses the history of a robot
bool comGraphEvent
int16[] robList
float32[] velocityList
History[] listOfH
